corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -0,0 +1,474 @@
+{% extends "style/bootstrap3/base_section.html" %}
+{% load i18n %}
+{% load hq_shared_tags %}
+{% load crispy_forms_tags %}
+{% load djangular_tags %}
+
+{% block js %}
+<script type="text/javascript"
+        src="{% new_static 'style/js/pagination.ng.js' %}"></script>
+<script type="text/javascript"
+        src="{% new_static 'users/js/mobile_workers.ng.js' %}"></script>
+{% endblock %}
+
+{% block js-inline %}{{ block.super }}
+<script type="text/javascript">
+$(function () {
+   'use strict';
+    $('#newMobileWorkerModal').on('shown.bs.modal', function () {
+        $('#id_username').focus();
+    });
+});
+(function (angular, undefined) {
+    'use strict';
+    var mobileWorkerApp = angular.module('mobileWorkerApp', ['hq.pagination', 'hq.mobile_workers']);
+    mobileWorkerApp.config(['$httpProvider', function($httpProvider) {
+        $httpProvider.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
+    }]);
+    mobileWorkerApp.config(["djangoRMIProvider", function(djangoRMIProvider) {
+        djangoRMIProvider.configure({% djng_current_rmi %});
+    }]);
+    mobileWorkerApp.constant('customFields', {{ custom_fields|JSON }});
+    mobileWorkerApp.constant('paginationLimitCookieName', '{{ pagination_limit_cookie_name }}');
+    mobileWorkerApp.constant('paginationCustomData', {
+        customFormFieldNames: {{ custom_field_names|JSON }},
+        showDeactivatedUsers: false,
+        activateUser: function (user, djangoRMI) {
+            $('#activate_' + user.user_id).modal('hide');
+            djangoRMI.modify_user_status({
+                user_id: user.user_id,
+                is_active: true
+            })
+                .success(function (data) {
+                    if (data.success) {
+                        user.mark_activated = true;
+                        user.action_error = '';
+                    } else {
+                        user.action_error = data.error;
+                    }
+                })
+                .error(function () {
+                    user.action_error = '{% trans "Issue communicating with server. Try again." %}';
+                });
+        },
+        deactivateUser: function (user, djangoRMI) {
+            $('#deactivate_' + user.user_id).modal('hide');
+            djangoRMI.modify_user_status({
+                user_id: user.user_id,
+                is_active: false
+            })
+                .success(function (data) {
```

Comment:
could abstract the success/error, but not sure it's totally necessary. if the logic gets any more complex maybe